hal/armv8m/mcx/n94x/config.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Phoenix-RTOS
+ *
+ * plo - operating system loader
+ *
+ * Platform configuration file
+ *
+ * Copyright 2022, 2024 Phoenix Systems
+ * Author: Damian Loewnau, Aleksander Kaminski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#ifndef _CONFIG_H_
+#define _CONFIG_H_
+
+#ifndef __ASSEMBLY__
+
+#include "n94x.h"
+#include "peripherals.h"
+#include "../types.h"
```

Comment:
In `plo` there also is top level `-I.` flag so I'd vote for moving to full path includes instead of relative, same as in kernel.